src/platform_impl/web/web_sys/canvas.rs

```diff
@@ -147,14 +138,6 @@ impl Canvas {
         }));
     }
 
-    pub fn on_touch_end(&mut self, prevent_default: bool) {
-        self.on_touch_end = Some(self.common.add_event("touchend", move |event: Event| {
-            if prevent_default {
-                event.prevent_default();
-            }
-        }));
-    }
```

Comment:
Same as above.
Apparently `preventDefault()` on `touchstart` was enough.

Reply:
Resolved in https://github.com/rust-windowing/winit/pull/2847#discussion_r1217186189.